microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md

```diff
@@ -0,0 +1,106 @@
+
+## Open the Detached Elements tool
+
+The **Detached Elements** tool is available as an experiment since Microsoft Edge version 93 and is available by default in Microsoft Edge version 97. Check your version of Microsoft Edge by navigating to `edge://version`. If your version number is less than 97, read [Turning an experiment on or off
+](/devtools-guide-chromium/experimental-features/#turning-an-experiment-on-or-off) for instructions on turning the `Detached Elements` experiment on.
+
+To open the **Detached Elements** tool against our demo application, complete the following steps.
+
+1. Navigate to the [demo application](https://microsoftedge.github.io/Demos/detached-elements/).
+1. [Open DevTools](/devtools-guide-chromium/#opening-devtools).
+1. Select the + button in the toolbar.
+1. Select the **Detached Elements** entry in the menu.
+
+:::image type="complex" source="./images/open-detached-elements.msft.png" alt-text="Open the Detached Elements tool." lightbox="./images/open-detached-elements.msft.png":::
+   Open the Detached Elements tool
+:::image-end:::
+
+## Get Detached Elements
+
+The **Get Detached Elements** button in the **Detached Elements** tool will find all of the detached elements on the page and display them. To find detached elements in our demo application, complete the following steps.
+
+1. Ensure you are in **Room 1** in the demo application.
+1. Select the **Simulate high traffic** button in the demo application.
+1. Select **Stop** after generating some messages in the demo application.
+1. Select **Room 2** in the demo application.
+1. Select **Get Detached Elements** in the **Detached Elements** tool.
+
+:::image type="complex" source="./images/get-detached-elements.msft.png" alt-text="Get Detached Elements in the Detached Elements tool." lightbox="./images/get-detached-elements.msft.png":::
+   Get Detached Elements in the Detached Elements tool
+:::image-end:::
+
+The **Detached Elements** tool displays a list of all of the detached elements on the page.  All of the messages that were generated in **Room 1** are no longer attached to the DOM when you switch to **Room 2** in the demo application but they are still referenced by JavaScript.  
+
+## Trigger Garbage Collection
+
+You can trigger garbage collection (GC) in the browser from the **Detached Elements** tool. Complete the following steps.
+
+1. Select the **Collect Garbage** button in the **Detached Elements** tool.
+1. Select the **Get Detached Elements** button in the **Detached Elements** tool.
+
+By selecting the **Collect Garbage** button, the browser will run GC, and by selecting the **Get Detached Elements** button, the **Detached Elements** tool will display all detached elements that cannot be GC'd. These detached elements may be memory leaks if they are not going to be reused by the application.
```

Comment:
@vchapel - "GC" should be ok if introduced spelled out.  But the construction "GD'd" is quirky, might need web research & judgment: how to introduce?  Is that construction viable per industry usage?
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/g/garbage-collection-garbage-collector

Reply:
I was just going to spell it out.